dashboard/test/ui/features/applab/level_options.feature

```diff
@@ -11,3 +11,19 @@ Scenario: Table data in level definition appears in data browser
   And I click selector "a:contains(table_name2)"
   And I wait until element "#dataTable" is visible
   And I wait until element "td:contains(Seattle)" is visible
+
+Scenario: Level defaults to design mode, students see design mode and teachers see code mode when viewing student work
+  #As student
+  Given I create a teacher-associated student named "Lillian"
+  And I am on "http://studio.code.org/s/csp5/stage/8/puzzle/9"
```

Comment:
Let's copy the relevant level into the `allthethings` script (or reuse one there) to avoid tests being broken by curriculum changes.